frontend/lib/src/components/core/Block/ElementNodeRenderer.tsx

```diff
@@ -566,10 +565,10 @@ const RawElementNodeRenderer = (
         <FileUploader
           key={fileUploaderProto.id}
           element={fileUploaderProto}
-          width={width}
           widgetMgr={widgetProps.widgetMgr}
           uploadClient={props.uploadClient}
           disabled={widgetProps.disabled}
+          {...elementProps}
```

Comment:
I believe we ca use `widgetProps` here and cut out the `widgetMgr` and `disabled` above